tests/cli/fluvio_smoke_tests/e2e-file-with-separator.bats

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bats
+
+TEST_HELPER_DIR="$BATS_TEST_DIRNAME/../test_helper"
+export TEST_HELPER_DIR
+
+load "$TEST_HELPER_DIR"/tools_check.bash
+load "$TEST_HELPER_DIR"/fluvio_dev.bash
+load "$TEST_HELPER_DIR"/bats-support/load.bash
+load "$TEST_HELPER_DIR"/bats-assert/load.bash
+
+setup_file() {
+    TOPIC_NAME=$(random_string)
+    export TOPIC_NAME
+    debug_msg "Topic name: $TOPIC_NAME"
+    
+    KEY1=$(random_string)
+    export KEY1
+    KEY2=$(random_string)
+    export KEY2
+    KEY3=$(random_string)
+    export KEY3
+    VAL1=$(random_string)
+    export VAL1
+    VAL2=$(random_string)
+    export VAL2
+    VAL3=$(random_string)
+    export VAL3
+    
+    SEPARATOR='||'
+    export SEPARATOR
+    
+    MULTI_LINE_FILE_CONTENTS=$KEY1$SEPARATOR$VAL1$'\n'$KEY2$SEPARATOR$VAL2$'\n'$KEY3$SEPARATOR$VAL3
+    export MULTI_LINE_FILE_CONTENTS
+    
+    MULTI_LINE_FILE_NAME=$(random_string)
+    export MULTI_LINE_FILE_NAME
+    
+    run bash -c 'echo "$MULTI_LINE_FILE_CONTENTS" > "$MULTI_LINE_FILE_NAME"'
+}
+
+teardown_file() {
+    run timeout 15s "$FLUVIO_BIN" topic delete "$TOPIC_NAME"
+    run rm $MULTI_LINE_FILE_NAME
+}
+
+# Create topic
+@test "Create a topic for file message with separator" {
+    if [ "$FLUVIO_CLI_RELEASE_CHANNEL" == "stable" ]; then
+        skip "don't run on stable version"
+    fi
```

Comment:
nice trick!

We should create an issue to update this once we release a new stable version

Reply:
thanks!